### DISCUSSION TITLE
Need to whitelist Budibase-Server to get Database Connection

### DISCUSSION THREAD

**Discussion by steffen-mzk** [Budibase/budibase#4451] [Help]
Hey there, 
Yesterday i tried to connect to a wordpress (mysql) database. I use Siteground as my hoster. I need to whitelist the servers ip from which i want to make remote access. 
could you give me the ip to whitelist it on my hoster? 
Therefore i can‘t connect to my database. 
Thanks

**Comment by SMetriq** [comment]
Hello, normally this is : 34.246.43.190

Fyi if you need, source : https://docs.budibase.com/building-apps/data

Regards
